projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1549,8 +1551,10 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * @hidden
      * @internal
      */
-    public handleOpening(event: CancelableEventArgs) {
-        this.onOpening.emit(event);
+    public handleOpening(event: CancelableBrowserEventArgs & IBaseEventArgs) {
```

Comment:
we have this union type `CancelableBrowserEventArgs & IBaseEventArgs` at many places now. Can we strongly type it in a new class, something like `BaseCancelableBrowseEventArgs` or something better?

Reply:
I'll introduce two of those - `IBaseCancelableEventArgs` and `IBaseCancelableBrowserEventsArgs`. It's in a separate commit, in case we decide it's not safe enough to merge in this PR.